rfcs/20240228-graph-api-support-f16-accumulation-mode/README.md

```diff
@@ -0,0 +1,214 @@
+# Graph API: Add Accumulation Mode Support in oneDNN Graph
+
+## Introduction
+
+Please refer to the [dcoumentation](https://oneapi-src.github.io/oneDNN/dev_guide_attributes_accumulation_mode.html#doxid-dev-guide-attributes-accumulation-mode) and [rfc](https://github.com/mgouicem/oneDNN/blob/mgouicem/rfcs/f16-accumulation/rfcs/20230118-f16-accumulation/README.md) for the motivation and design detail of primitive API.
+
+Currently, oneDNN uses f32 for floating point computation and s32 for integer computation as the default accumulation data types. However, on some platforms, using smaller accumulation data types can result in additional speed improvements.
+
+By introducing support for the accumulation data type, oneDNN can achieve up to a 2x speedup while maintaining a high level of accuracy for f16 inference. It is important to note that f16 accumulation is not suitable for training, which requires full f32 precision.
```

Comment:
Based on my understanding, the speedup comes from the lower-precision data type's reduced computational and memory requirements. 
In terms of platform requirements, currently, this feature will mainly be applied in f16 inference scenarios, so it requires the device to support f16 on CPU. And it will be applied to the computation-intensive operations such as `convolution` and `matmul`.